python/nano/benchmark/README.md

```diff
@@ -0,0 +1,44 @@
+# BigDL Nano Benchmark (in progress)
+
+This directory contains workloads used for bigdl-nano preformance regression test, this document shows how to add new workloads:
+
+## Directory Structure
+
+Every subdirectory under the current directory is a workload, the name of subdirectory is the name of workload.
+
+Every subdirectory should contain a `run.sh`, which is used for running workload.
+
+No other rules to follow.
```

Comment:
how about specifying the output format of performance numbers?

Reply:
have not decided yet

Reply:
maybe we also need a way to specify the argument config? i.e. use_ipex, num_processes, etc.